### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by Nick5500**

### CODE REVIEW

File: authentication/src/app.module.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { SequelizeModule } from '@nestjs/sequelize';
+import { UserModule } from './user/user.module';
+import { AuthModule } from './auth/auth.module';
+
+@Module({
+  imports: [
+    SequelizeModule.forRoot({
+      dialect: 'postgres',
+      host: 'localhost',
+      port: 5433,
```
Comment (MaxLogvyniukFive) [inline authentication/src/app.module.ts:11]: Use value from .env

File: authentication/src/app.module.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Module } from '@nestjs/common';
+import { SequelizeModule } from '@nestjs/sequelize';
+import { UserModule } from './user/user.module';
+import { AuthModule } from './auth/auth.module';
+
+@Module({
+  imports: [
+    SequelizeModule.forRoot({
+      dialect: 'postgres',
+      host: 'localhost',
```
Comment (MaxLogvyniukFive) [inline authentication/src/app.module.ts:10]: Use value from .env

File: authentication/src/app.module.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { Module } from '@nestjs/common';
+import { SequelizeModule } from '@nestjs/sequelize';
+import { UserModule } from './user/user.module';
+import { AuthModule } from './auth/auth.module';
+
+@Module({
+  imports: [
+    SequelizeModule.forRoot({
+      dialect: 'postgres',
```
Comment (MaxLogvyniukFive) [inline authentication/src/app.module.ts:9]: Use value from .env

File: authentication/src/app.module.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      host: 'localhost',
+      port: 5433,
+      username: 'postgres',
```
Comment (MaxLogvyniukFive) [inline authentication/src/app.module.ts:12]: Use value from .env for all values

File: authentication/src/auth/auth.service.ts [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+    return { access_token: tokens.access_token };
+  }
+
+  generateTokens(payload) {
+    const access_token = this.jwtService.sign(
+      {
+        email: payload.email,
+        sub: payload.id,
+      },
+      { secret: process.env.ACCESS_SECRET, expiresIn: '15m' },
```
Comment (MaxLogvyniukFive) [inline authentication/src/auth/auth.service.ts:48]: 15m move in variable

File: authentication/src/auth/auth.service.ts [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+    );
+    const refresh_token = this.jwtService.sign(
+      { sub: payload.id },
+      { secret: process.env.REFRESH_SECRET, expiresIn: '30m' },
```
Comment (MaxLogvyniukFive) [inline authentication/src/auth/auth.service.ts:52]: Same here

File: authentication/src/app.module.ts (hunk 1 above)
Comment (Nick5500) [reply authentication/src/app.module.ts:11]: dialect can't be pasted from .env. It requires value directly

**Pull request closed without merging**